frontend/src/components/listOfProducts/ListHeader.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+const ListHeader = () => {
+  const { t } = useTranslation()
+
+  return (
+    <thead>
+      <tr>
+        <th scope="col">{t('product.name')}</th>
+        <th scope="col">{t('product.address')}</th>
+        <th scope="col">{t('product.city')}</th>
+        <th scope="col">{t('product.category')}</th>
+        <th scope="col">{t('product.owner')}</th>
+        <th>{t('')}</th>
```

Comment:
Ha ez üres kell, hogy legyen, akkor ne fordítsuk le. Akkár ki is maradhat: `<th />`